full-service/src/db/txo.rs

```diff
@@ -15,22 +15,55 @@ use mc_transaction_core::{
     tx::{TxOut, TxOutConfirmationNumber},
     Amount, Token,
 };
-use std::fmt;
+use std::{fmt, str::FromStr};
 
 use crate::{
     db::{
         account::{AccountID, AccountModel},
         assigned_subaddress::AssignedSubaddressModel,
-        models::{
-            Account, AssignedSubaddress, NewTxo, Txo, TXO_STATUS_ORPHANED, TXO_STATUS_PENDING,
-            TXO_STATUS_SECRETED, TXO_STATUS_SPENT, TXO_STATUS_UNSPENT,
-        },
-        transaction_log::TransactionID,
+        models::{Account, AssignedSubaddress, NewTransactionOutputTxo, NewTxo, Txo},
+        transaction_log::{TransactionID, TxoType},
         Conn, WalletDbError,
     },
     util::b58::b58_encode_public_address,
 };
 
+#[derive(Debug, PartialEq)]
+pub enum TxoStatus {
+    Unverified,
+    Unspent,
+    Pending,
+    Spent,
+    Orphaned,
+}
+
+impl fmt::Display for TxoStatus {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            TxoStatus::Unverified => write!(f, "unverified"),
```

Comment:
maybe not necessary for this PR, but I think a commented description for each txo status would be super helpful. Things to include: high-level description of what this status means, DB conditions for this status

Reply:
will add! definitely useful and should be included :)